creator-node/src/config.js

```diff
@@ -402,6 +402,12 @@ const config = convict({
     env: 'discoveryProviderWhitelist',
     default: ''
   },
+  discoveryNodeUnhealthyBlockDiff: {
+    doc: 'Number of missed blocks after which a discovery node would be considered unhealthy',
+    format: 'nat',
+    env: 'discoveryNodeUnhealthyBlockDiff',
+    default: 500
```

Comment:
sry for drive-by comment (not blocker for merging!). i know this isn't a change in this PR, but I'd be curious to know the implications of a "behind" discovery node a bit more deeply. I know we no longer us the discovery node to verify what the user's replica set is in order to allow/block writes from the primary, which was AFAIK the most critical path, but 500 blocks is still 41 minutes. Client-side, we generally use 15 blocks as the proxy. In cases where *every* discovery node is behind by 15 blocks, that's obviously bad, but that might not be super destructive if we don't depend on discovery node for super critical path stuff.

cc @dmanjunath

Reply:
the rationale to add this was in the event all discovery nodes were behind by > 15 blocks. if a content node ever went down it would never come back up because it could never initialize libs so it will be in CrashLoopBackOff. That means whatever content it have can't be served, so discovery can't progress and the cycle continues. 

i'm not opposed to lowering this limit but the tradeoff is it doesn't boot up in the event discovery falls behind. the reason i'm okay with this is our selection logic should still select highest version, lowest blockdiff anyway in the happy path right @raymondjacobson?

Reply:
We should look into that separately. Libs should support " regressed " mode when no discovery nodes are found that are behind by a healthy amount. queries, etc. will work to the node, but it just signifies that nothing better could be found.

https://github.com/AudiusProject/audius-protocol/blob/17cc1d235208aa5237db4489b69defa4c581aaae/libs/src/services/discoveryProvider/DiscoveryProviderSelection.js#L43

Reply:
if i understand you correctly that libs should fall back to a regressed discovery node i completely agree. content node is read only so not super critical, but if every discovery node is behind we have bigger problems lol

Reply:
is 500 the desired block diff number for now?